doc/conf.py

```diff
@@ -84,19 +80,20 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "sphinx_rtd_theme"
+html_theme = 'pydata_sphinx_theme'
+
+# The Xnd logo
+html_logo = "images/xndlogo.png"
 
 html_theme_options = {
-    # 'html_baseurl': 'https://xnd.io/',
-    'analytics_id': '',
-    'logo_only': False,
-    'display_version': True,
-    'prev_next_buttons_location': 'bottom',
-    # Toc options
-    'collapse_navigation': True,
-    'sticky_navigation': True,
-    # 'navigation_depth': 4,
+    "logo_link": "index",
+    "github_url": "https://github.com/quansight/rbc",
```

Comment:
just a note: this does not exist yet

Reply:
Yeah I will revert this (and bellow for the `html_context`) quick so we can merge and I can change it as soon as we moved the repo. Same for the logo actually, I suppose we should put Quansight's one once we move.

When we do the move we should also take the opportunity to rename the branch to `main`. BTW, who is doing this? I don't think I have permissions to help on that.

Reply:
Before moving, let's have one rbc release first.
^ @guilhermeleobas 
Re main branch, see https://github.com/xnd-project/rbc/issues/328